packages/interface-adapter/package.json

```diff
@@ -19,16 +19,17 @@
     "watch": "tsc -w"
   },
   "dependencies": {
+    "@truffle/blockchain-utils": "^0.0.13",
     "bn.js": "^4.11.8",
     "ethers": "^4.0.32",
     "lodash": "^4.17.13",
-    "web3": "1.2.1"
+    "web3": "1.2.2",
+    "web3-eth": "1.2.2"
```

Comment:
@CruzMolina I *think* when web3 is installed, all the subsidiary web3- packages are auto-populated to `node_modules`. But in some truffle packages (truffle/contract is an example), some sub-packages are also individually listed as deps. 

For purely selfish reasons (an E2E test at web3 which uses Truffle as a target and needs to swap web3 in/out), it would be convenient if this pattern was normalized a bit. 

Is this possible or are you finding it necessary to have the separate dep?

Reply:
IIRC I started adding sub-packages this way at truffle/contract because an early 1.x beta suddenly changed the way everything was published, and then reversed course. But I lost track of it...

Reply:
Let me look into. I _might_ need to specify the separate dep for typings, but I can probably adjust the `tsconfig`'s to find the sub-pkgs.

Reply:
@CruzMolina Ok cool - yeah if it's not convenient for you please just ignore.

Reply:
Pushed the changes, seems to work locally 🤞